indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -657,6 +669,37 @@ public String getType()
     }
   }
 
+  private class HandoffTaskGroupNotice implements Notice
+  {
+    final Integer taskGroupId;
+    private static final String TYPE = "handoff_task_group_notice";
+
+    HandoffTaskGroupNotice(
+        final Integer taskGroupId
```

Comment:
Add Nonnull since it should be validated elsewhere

```suggestion
        @Nonnull final List<Integer> taskGroupIds
```